src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -172,28 +168,6 @@ private static boolean continueConsuming(final int messagesConsumed, final int m
     produceKeyValuesSynchronously(topic, keyedRecords, producerConfig);
   }
 
-  /**
-   * Like {@link IntegrationTestUtils#produceValuesSynchronously(String, Collection, Properties)}, except for use with
-   * TopologyTestDriver tests, rather than "native" Kafka broker tests.
-   *
-   * @param topic              Kafka topic to write the data records to
-   * @param values             Message values to write to Kafka (keys will have type byte[] and be set to null)
-   * @param topologyTestDriver The {@link TopologyTestDriver} to send the data records to
-   * @param valueSerializer    The {@link Serializer} corresponding to the value type
-   * @param <V>                Value type of the data records
-   */
-  static <K, V> void produceValuesSynchronously(final String topic,
```

Comment:
With the new KIP-470 API, we don't need those helpers any longer.

Reply:
Yay!